bin/universe.sh

```diff
@@ -55,28 +58,37 @@ fetch_commons_tools() {
     fi
 }
 
-make_cli() {
-    CLI_VERSION="${CLI_VERSION}" make --directory=cli all
+build_cli() {
+    echo "build_cli: start"
+    pwd
+    ls ${SPARK_BUILD_DIR}/cli/
+    make --directory=cli all
 }
 
 make_docker() {
+    echo "make_docker: start"
     (cd "${SPARK_BUILD_DIR}" && DOCKER_IMAGE=${DOCKER_IMAGE} make docker)
 }
 
 upload_cli_and_stub_universe() {
+    echo "upload_cli_and_stub_universe: start"
     # Build/upload package using custom template parameters: TEMPLATE_X_Y_Z => {{x-y-z}}
     TEMPLATE_CLI_VERSION=${CLI_VERSION} \
     TEMPLATE_SPARK_DIST_URI=$(default_spark_dist) \
     TEMPLATE_DOCKER_IMAGE=${DOCKER_IMAGE} \
         ${COMMONS_TOOLS_DIR}/ci_upload.py \
             spark \
-            ${DISPATCHER_DIR}/package/ \
-            ${DISPATCHER_DIR}/cli/dist/*.whl
+            ${SPARK_BUILD_DIR}/universe/ \
+            ${SPARK_BUILD_DIR}/cli/dcos-spark/dcos-spark-darwin \
+            ${SPARK_BUILD_DIR}/cli/dcos-spark/dcos-spark-linux \
+            ${SPARK_BUILD_DIR}/cli/dcos-spark/dcos-spark.exe \
+            ${SPARK_BUILD_DIR}/cli/python/dist/*.whl
```

Comment:
Actually, per offline discussion, it'd make sense to keep the python wrapper for now as it's needed for compatibility with spark-build's old tooling that's still on universe-2.x.